### ISSUE TITLE
ground truth boxes format and transforms 

### ISSUE THREAD

**Issue by neeek2303** [dbolya/yolact#460]
Hello
I am trying to train yolact on a custom dataset and didn't get which format of ground truth boxes MultiBoxLoss wait. I try to put boxes with coordinates in pixels (for example 307,  255,  602,  416, for 640x1024 picture), but it doesn't work. Could you please clarify what type of boxes MultiBoxLoss wait.

**Comment by PrajwalCogniac** [comment]
Hey @neeek2303 
I am having an issue where I already have the ground truth masks of shape (1,h,w) they are binary masks then how do I modify/ use this loss function ?
